### ISSUE TITLE
Paper Path

### ISSUE THREAD

**Issue by johnren-code** [PuTeGuo/GeoSelfSSC-main#1]
Hello, may i ask your paper path?

**Comment by PuTeGuo** [comment]
Hello! Thank you for your interest in this work. The paper is currently under review. Once it is accepted, I will update this page with the article link.

**Comment by johnren-code** [comment]
OK, Thank you very much!

**Comment by PuTeGuo** [comment]
Hello! The pre-trained weights provided by Panoptic-DeepLab have been removed, so we recommend contacting the original author for access. In our experiments, to ensure a fair comparison and validate the effectiveness of our work, we used the existing segmentation pseudo-labels provided by S4C for training.
